### PR TITLE
Fixing squid:S1488 and squid:S3973

### PR DESCRIPTION
**Pull request by warlockza**

### CODE REVIEW

File: src/main/java/org/cloudfoundry/promregator/cfaccessor/CacheKeyApplication.java [hunk 1]
```diff
@@ -59,18 +59,21 @@ public boolean equals(Object obj) {
 		if (applicationName == null) {
 			if (other.applicationName != null)
 				return false;
-		} else if (!applicationName.equals(other.applicationName))
+		} else if (!applicationName.equals(other.applicationName)) {
```
Comment (eaglerainbow) [inline src/main/java/org/cloudfoundry/promregator/cfaccessor/CacheKeyApplication.java]: see also https://github.com/promregator/promregator/pull/129/files#r352367615

File: src/main/java/org/cloudfoundry/promregator/cfaccessor/CFAccessorCacheClassic.java [hunk 2]
```diff
@@ -1,396 +1,392 @@
-package org.cloudfoundry.promregator.cfaccessor;
```
Comment (eaglerainbow) [inline src/main/java/org/cloudfoundry/promregator/cfaccessor/CFAccessorCacheClassic.java]: You converted the entire file from newline format `lf` (which is the default for Promregator) to `cflf`. This isn't good 😬

Comment (warlockza) [reply src/main/java/org/cloudfoundry/promregator/cfaccessor/CFAccessorCacheClassic.java]: Yes, have been having some wierd issues. Should be fixed now

**Comment by eaglerainbow** [comment]
BTW: CI job is just running (I have to still trigger them manually - many reasons...).
We should see an update in [SonarCloud](https://sonarcloud.io/dashboard?id=promregator) soon.

**Comment by eaglerainbow** [comment]
Finished....
Fixing the old one's caused new Sonar violations to appear...

**Pull request merged**